### PR TITLE
Removed forced exclusions of fields (..0)

### PR DESCRIPTION
**Pull request by peturv**
Added direkt link to search for commonprofile builds in zulip

### CODE REVIEW

**Review by vadi2 — COMMENTED** [review]
What about ServiceRequestSelfReferralSEVendorLite (that's a mouthful!) as well?

File: README.md [hunk 1]
```diff
@@ -49,4 +50,4 @@ https://commonprofiles.care/StructureDefinition/common-hsaHierarchy
 
 Including the resource type follows the same pattern which is used both for the Swedish base profiles and also for the FHIR base resources and profiles. Examples:
 http://build.fhir.org/ig/HL7Sweden/basprofiler-r4/StructureDefinition-SEBaseOrganization.html
-https://www.hl7.org/fhir/bodyweight.html
\ No newline at end of file
+https://www.hl7.org/fhir/bodyweight.html
```
Comment (MangalaEkanayake) [inline README.md:52]: I wonder you made some changes this line?

File: README.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ Published destination: https://commonprofiles.care/
 More info
 * https://github.com/FHIR/auto-ig-builder/blob/master/README.md
 * https://chat.fhir.org/#narrow/stream/179297-committers.2Fnotification/topic/ig-build
+* https://chat.fhir.org/#narrow/stream/179297-committers.2Fnotification/topic/ig-build/search/svenskaindustriprofiler.2Ffhir
```
Comment (MangalaEkanayake) [inline README.md:7]: It is very good that we redirect to the specific page.
Now we need a link to https://chat.fhir.org/#narrow/stream/179297-committers.2Fnotification/topic/ig-build?
Can't we remove line 7 now?

File: README.md (hunk 1 above)
Comment (peturv) [reply README.md:52]: > I wonder you made some changes this line?

No, just a whitespace change

File: README.md (hunk 2 above)
Comment (peturv) [reply README.md:7]: > It is very good that we redirect to the specific page. Now we need a link to https://chat.fhir.org/#narrow/stream/179297-committers.2Fnotification/topic/ig-build? Can't we remove line 7 now?

Fixed!

**Review by niclas86 — APPROVED** [review]

**Review by vjernelov — APPROVED** [review]
Looks solid!

**Comment by peturv** [comment]
> What about ServiceRequestSelfReferralSEVendorLite (that's a mouthful!) as well?

I had missed this. Has now been updated in according to the 'don't be excluding' strategy.

**Review by MangalaEkanayake — APPROVED** [review]

**Pull request merged**